util/libcrypto.num

```diff
@@ -4796,3 +4796,27 @@ EVP_MD_fetch                            4743	3_0_0	EXIST::FUNCTION:
 EVP_set_default_properties              4744	3_0_0	EXIST::FUNCTION:
 OSSL_PARAM_construct_end                4745	3_0_0	EXIST::FUNCTION:
 EC_GROUP_check_named_curve              4746	3_0_0	EXIST::FUNCTION:EC
+OSSL_CMP_PKISTATUS_it                   4747	3_0_0	NOEXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_PKISTATUS_it                   4748	3_0_0	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_PKISTATUS_it                   4748	3_0_0	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:CMP
+d2i_OSSL_CMP_PKIHEADER                  4749	3_0_0	EXIST::FUNCTION:CMP
+i2d_OSSL_CMP_PKIHEADER                  4750	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_PKIHEADER_free                 4751	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_PKIHEADER_new                  4752	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_PKIHEADER_it                   4753	3_0_0	NOEXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_PKIHEADER_it                   4754	3_0_0	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_PKIHEADER_it                   4754	3_0_0	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:CMP
+d2i_OSSL_CMP_MSG                        4755	3_0_0	EXIST::FUNCTION:CMP
+i2d_OSSL_CMP_MSG                        4756	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_MSG_it                         4757	3_0_0	NOEXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_MSG_it                         4758	3_0_0	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_MSG_it                         4758	3_0_0	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:CMP
+OSSL_CMP_ITAV_set0                      4759	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_ITAV_get0_type                 4760	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_ITAV_get0_value                4761	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_ITAV_stack_item_push0          4762	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_ITAV_free                      4763	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_MSG_free                       4764	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_PKISI_free                     4765	3_0_0	EXIST::FUNCTION:CMP
+OSSL_CMP_MSG_dup                        4766	3_0_0	EXIST::FUNCTION:CMP
+ERR_load_CMP_strings                    4767	3_0_0	EXIST::FUNCTION:CMP
```

Comment:
All the above functions should appear in the docs somewhere

Reply:
Done as described above.

Note that so far none of the many `ERR_load_<lib>_strings` functions are explicitly documented apart from some general description in `ERR_load_strings`. So I leave this to some general OpenSSL crypto and ssl lib maintenance.